packages/graphql-language-service-server/src/MessageProcessor.js

```diff
@@ -107,8 +113,15 @@ export class MessageProcessor {
       );
     }
 
-    this._graphQLCache = await getGraphQLCache(rootPath);
-    const config = getGraphQLConfig(rootPath);
+    this._graphQLCache = await getGraphQLCache(rootPath, this._extensions);
+    let config = getGraphQLConfig(rootPath);
+    if (this._extensions && this._extensions.length > 0) {
+      /* eslint-disable no-await-in-loop */
+      for (let i = 0; i < this._extensions.length; i++) {
```

Comment:
we have support for the latest language features now, so if you want to use for... await of here you can! I think Its node 9+.

Reply:
Bumped the loop to new syntax but kept `/* eslint-disable no-await-in-loop */` becuase lint still disables it.

Reply:
@divyenduz interesting note, perhaps we should change this rule now that we have modern for...await of